docs/getting-started-guides/ubuntu/upgrades.md

```diff
@@ -141,7 +141,7 @@ to ensure that the cluster upgrade has successfully completed.
 ### Upgrade Flannel
 
 Upgrading flannel can be done at any time, it is independent of Kubernetes upgrades.
-Be advised that networking is interrupted during the upgrade. You can initiate a flannel upgrade:
+Be advised that networking is interrupted during the upgrade:
```

Comment:
The old text reads better to me.

Reply:
No problem, I will revert the change. I forgot to motivate the change: In the paragraph below about the easyrsa upgrade, there is no such introductory sentence.